utilities_win32.cpp

```diff
@@ -461,7 +461,7 @@ bool Utilities::isSystemMenuRequested(const void *data, QPointF *pos)
     }
     bool result = false;
     const auto msg = static_cast<const MSG *>(data);
-    if (msg->message == WM_NCRBUTTONUP) {
+    if (msg->message == WM_NCRBUTTONDOWN ) {
```

Comment:
Doesn't this mean the system menu will instantly show up when the user pressed the right button on the title bar? This doesn't seem correct.

Reply:
ping?
Other parts of this PR look good.

Reply:
Sorry I did not see your comment. You are correct the menu will show when pressed and not released.
If you tried without this line?

Reply:
I've not tried, but the code looks not wrong, except this line. Why do you change this line?

Reply:
I just tried, it doesn't work if you monitor UP instead of DOWN, this is due to https://github.com/wangwenx190/framelesshelper/pull/99/files
On the other hand, I reverted #99 and it seems the bug when closing menus/comboboxes is gone. It would need further testing though